### PR TITLE
Feature 591 csm-installation-wizard documentation page update

### PR DESCRIPTION
**Pull request by niranjan-n1**
# Description
Updated the csm-installation-wizard documentation page.
# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| dell/csm#591 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/deployment/csminstallationwizard/index.md [hunk 1]
```diff
@@ -30,18 +30,52 @@ The [Dell Container Storage Modules Installation Wizard](./src/index.html) is a
...
+
+>> NOTE: Only the Authorization sidecar is enabled by the CSM Installation Wizard. The Proxy Server has to be installed and configured separately.
+
+6. If the Volume Snapshot feature is enabled, please refer to [Volume Snapshot for PowerStore](../../csidriver/installation/helm/powerstore/#optional-volume-snapshot-requirements) and [Volume Snapshot for PowerMax](../../csidriver/installation/helm/powermax/#optional-volume-snapshot-requirements) to install the Volume Snapshot CRDs and the default snapshot controller.
+
+>> NOTE: The CSM Installation Wizard generates values.yaml with the minimal inputs required to install the CSM. To configure additional parameters in values.yaml, please follow the steps outlined in [PowerStore](../../csidriver/installation/helm/powerstore/#install-the-driver), [PowerMax](../../csidriver/installation/helm/powermax/#install-the-driver), [Observability](../../observability/), [Replication](../../replication/), [Resiliency](../../resiliency/), and [Application Mobility](../../applicationmobility/).
+
+7. Install the Helm chart.
+
+    On your terminal, run the below command:
```
Comment (rsedlock1958) [inline content/docs/deployment/csminstallationwizard/index.md:77]: Replace "below" with "this"

**Review by rsedlock1958 — DISMISSED** [review]
I have reviewed this. One minor edit is required.

File: content/docs/deployment/csminstallationwizard/index.md (hunk 1 above)
Comment (niranjan-n1) [reply content/docs/deployment/csminstallationwizard/index.md]: Addressed. Thanks

**Review by rsedlock1958 — APPROVED** [review]
I have reviewed this. No edits are required.

File: content/docs/deployment/csminstallationwizard/index.md [hunk 2]
```diff
@@ -30,18 +30,52 @@ The [Dell Container Storage Modules Installation Wizard](./src/index.html) is a
...
+6. If the Volume Snapshot feature is enabled, please refer to [Volume Snapshot for PowerStore](../../csidriver/installation/helm/powerstore/#optional-volume-snapshot-requirements) and [Volume Snapshot for PowerMax](../../csidriver/installation/helm/powermax/#optional-volume-snapshot-requirements) to install the Volume Snapshot CRDs and the default snapshot controller.
+
+>> NOTE: The CSM Installation Wizard generates values.yaml with the minimal inputs required to install the CSM. To configure additional parameters in values.yaml, please follow the steps outlined in [PowerStore](../../csidriver/installation/helm/powerstore/#install-the-driver), [PowerMax](../../csidriver/installation/helm/powermax/#install-the-driver), [Observability](../../observability/), [Replication](../../replication/), [Resiliency](../../resiliency/), and [Application Mobility](../../applicationmobility/).
+
+7. Install the Helm chart.
+
+    On your terminal, run this command:
+
+    ```terminal
+    helm install container-storage-modules dell/container-storage-modules -f <values.yaml location>
```
Comment (chaganti-rajitha) [inline content/docs/deployment/csminstallationwizard/index.md:80]: Please change release name and also mention in the note that all the secrets should be pre-created.

Comment (niranjan-n1) [reply content/docs/deployment/csminstallationwizard/index.md]: @chaganti-rajitha  Note regarding secrets is already there before the step 1.
Changed the release name. Thanks

**Review by chaganti-rajitha — APPROVED** [review]

**Pull request merged**